client/my-sites/plans/main.jsx

```diff
@@ -144,29 +144,27 @@ class Plans extends React.Component {
 						/>
 					) }
 					{ canAccessPlans && (
-						<div id="plans" className="plans plans__has-sidebar">
-							<FormattedHeader
-								className="plans__page-heading"
-								headerText={ translate( 'Plans' ) }
-								align="left"
-							/>
-							<CartData>
-								<PlansNavigation path={ this.props.context.path } />
-							</CartData>
-							<PlansFeaturesMain
-								displayJetpackPlans={ displayJetpackPlans }
-								hideFreePlan={ true }
-								customerType={ this.props.customerType }
-								intervalType={ this.props.intervalType }
-								selectedFeature={ this.props.selectedFeature }
-								selectedPlan={ this.props.selectedPlan }
-								redirectTo={ this.props.redirectTo }
-								withDiscount={ this.props.withDiscount }
-								discountEndDate={ this.props.discountEndDate }
-								site={ selectedSite }
-								plansWithScroll={ false }
-							/>
-						</div>
+						<>
+							<FormattedHeader headerText={ translate( 'Plans' ) } align="left" />
```

Comment:
Did we intentionally drop `className="plans__page-heading"` ?

Reply:
Yes, I noted it was not used at all, so decided to remove it.